static/auras.js

```diff
@@ -5,6 +5,7 @@
 		event.preventDefault();
 		const account = form.querySelector('input[name=account]').value;
 		const character = form.querySelector('input[name=character]').value;
-		window.location = `/auras/${account}/${character}`;
+		const aura_effect = form.querySelector('input[name=aura_effect]').value;
+		window.location = `/auras/${account}/${character}/${aura_effect}`;
```

Comment:
how about
```suggestion
		window.location = `/auras/${account}/${character}?aura_effect=${aura_effect}`;
```
and then you can
```py
if 'aura_effect' in request.params:
    aura_effect = int(request.params['aura_effect'])
```
and remove the param from the `routes`